staging/src/k8s.io/apiserver/pkg/endpoints/filters/traces.go

```diff
@@ -45,3 +54,26 @@ func WithTracing(handler http.Handler, tp trace.TracerProvider) http.Handler {
 	// See https://github.com/open-telemetry/opentelemetry-go/tree/main/example/passthrough
 	return otelhttp.NewHandler(wrappedHandler, "KubernetesAPI", opts...)
 }
+
+func getSpanNameFromRequestInfo(info *request.RequestInfo, r *http.Request) string {
+	if !info.IsResourceRequest {
+		return r.Method + " " + info.Path
```

Comment:
Is it safe to include the path here?  We are supposed to make sure this isn't high cardinality per https://github.com/kubernetes/kubernetes/pull/123584/files#r1523347214

Reply:
You could just check:

```golang
if !exist || !info.IsResourceRequest
```

above

Reply:
You are right, the requests with resource info is deserved to be parsed.

Reply:
Done